package.json

```diff
@@ -70,4 +72,4 @@
       "optional": true
     }
   }
-}
+}
```

Comment:
missing a new line at the end of the file. Probably my fault when i was resolving conflicts